### ISSUE TITLE
Request: View comment replies without JS

### ISSUE THREAD

**Issue by BurnyBoi** [omarroth/invidious#409]
Currently, it seems that viewing comment replies requires JS. Would it be possible to make the View replies button work without JavaScript? It is sensible that a new page would have to load. Maybe it could load the comment thread as it's own page so the user can just open it in a new tab?

**Comment by BurnyBoi** [comment]
My apologies. I see this is already being tracked under https://github.com/omarroth/invidious/issues/341

**Comment by omarroth** [comment]
I was planning on leaving a comment in #204 with this exact proposal. It seems wasteful to me to reload the `/watch` page multiple times in order to view comments without JS, and it's difficult to load all comments on initial page load since each thread is a separate request.

If you would like, I think it is sensible to leave this open as a request to add a separate `/comment_viewer`  page or similar, and track this in #341.

I would also appreciate feedback on this from @Jtasiu.

**Comment by BurnyBoi** [comment]
Agreed. Loading the watch page over and over again is wasteful for any feature. Having a separate comments page would be great; not only for loading replies without JS, but also for being able to view pages of comments without JS. Currently one can only load the top 10 comments with JS disabled if my understanding is correct.

**Comment by Jtasiu** [comment]
@omarroth

In my opinion, the best way to do this is to load the top comments with the watch page (like it is now) but if the user wants to view more comments there could be a "View all comments" link at the bottom of the page which would open a new page with all the comments. 

Also if it can be configured by default so that the top 50 comments are loaded with the watch page when JS is off instead of the current 10 it would be better, so that opening a separate comments page/viewer is less frequently needed. Honestly, the comments load so fast without JS that I don't even notice any delay. In fact with JS on for some reason comments takes even longer to load, I always see the progress spinner for a few seconds before comments load (maybe something to do with replies loading ability or loading comments differently when JS is on but I don't know) .

**Comment by omarroth** [comment]
Might make sense to rename this to "Request: comment viewer without JS". Added a link to track in #341.

**Comment by nachanon** [comment]
@omarroth; in some eons ago, YouTube provided a page which its sole content being _all comments_ posted on a specific video...

	https://www.youtube.com/all_comments?v=VIDEO_ID

It won't work anymore now; but I still have this URL pattern in my browser's search plugin (replace `VIDEO_ID` with the usual YouTube video ID) from the days this was still working.

So, I would suggest that this to-be-made "All comments" viewing page be done in the compatible URI scheme with the YouTube's old one.

**Comment by syeopite** [comment]
Bump

**Comment by nvidiaLinuxUser** [comment]
+1. Many people use the internet without JavaScript, I have to turn on JS to see comment replies.
